go/teams/service_helper.go

```diff
@@ -319,11 +226,16 @@ func userVersionsToDetails(mctx libkb.MetaContext, uvs []keybase1.UserVersion, t
 			}
 			fullName = pkg.FullName.FullName
 		}
+		role, err := t.chain().GetUserRole(uv)
+		if err != nil {
+			return nil, err
+		}
 		ret[i] = keybase1.TeamMemberDetails{
 			Uv:       uvs[i],
 			Username: pkg.NormalizedUsername.String(),
 			FullName: fullName,
 			Status:   status,
+			Role:     role,
 		}
 		if status == keybase1.TeamMemberStatus_ACTIVE && t != nil {
 			joinTime, err := t.UserLastJoinTime(uv)
```

Comment:
so while we are here, can we figure out duplication of 
```
func (t TeamSigChainState) MemberCtime(user keybase1.UserVersion) *keybase1.Time
```
```
func (s TeamSigChainState) GetUserLastJoinTime(user UserVersion) (time Time, err error)
```
?

IIRC it's has the same function, so we can just do with one.

Reply:
Ticket

Reply:
(it seems not immediately obvious)